README.md

```diff
@@ -1,38 +1,92 @@
+[![Build Status](https://travis-ci.org/facn2/Path18.svg?branch=master)](https://travis-ci.org/facn2/Path18)
+[![codecov](https://codecov.io/gh/facn2/Path18/branch/master/graph/badge.svg)](https://codecov.io/gh/facn2/Path18)
+
+**[Web app](https://path18.herokuapp.com/) for informing young people of a variety of careers and paths** :mortar_board:
+
+### Local Install
+
+Clone the repository.
+```
+git clone https://github.com/facn2/Path18.git
+```
+Install npm modules.
+```
+npm install
+```
+View on localhost.
+```
+npm start
+```
+Run tests.
+```
+npm test
+```
 
 ### Problem Statement:
-* Unexposure to variety of jobs and career paths for young people
-* Uncertainty around career prospects
+* Young people have little exposure to the variety of jobs and careers paths they could pursue.
+* Uncertainty around career prospects.
+* Little knowledge of the path to take to get a specific job.
 
 ### User Journey:
 As a user I can browse a variety of career paths and view detailed information about the ones that I am interested in.
 
 ### User Story:
 ##### As a user I can...
-* read welcome page and instructions on how to use the app
-* view card with job and decide if I like / dislike it
-* click on card to view more details
-* View list of jobs I like
-* click on job to view details on the job and career path
-* read details instructions for the app in the help page
+* read welcome page and instructions on how to use the app.
+* view card with job and decide if I like / dislike it.
+* click on card to view more details.
+* View list of jobs I like.
+* click on job to view details on the job and career path.
+* read details instructions for the app in the help page.
+
+---
+
+### Prototype:
 
-### Wireframe:
+Take a look at the prototype on [figma](https://www.figma.com/file/Mm2muc8uV1Km48foBZcntrEi/Education-App).
+
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076462-ddb5dd62-a782-11e7-993b-b03bd4b87c79.png" width=49%>
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076752-61ce4a70-a784-11e7-8542-998d451edd45.png" width=50%>
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076464-dde5ca54-a782-11e7-9d93-30a080092e30.png" width=25%>
+<img alt="Prototype start and info page" src="https://user-images.githubusercontent.com/25408167/31076466-ddead8dc-a782-11e7-8e45-ea310e21ac7a.png" width=25%>
 
 ### Architecture:
 
+![App architecture](https://user-images.githubusercontent.com/25408167/31123917-e141ee08-a84a-11e7-8524-57ca1ba9e4a5.jpg)
+
 ### Technologies:
+* Express
+* Handlebars
+* MongoDB
+* Mocha
+* Chai
 
 ### Team Guidelines:
 
-* CamelCase for JavaScript id's
-* kebab-case for CSS Classes
-* comments for code clarification
-* ES6 on backend
-* all team member understand code before approval
+* CamelCase for JavaScript id's.
+* kebab-case for CSS Classes.
```

Comment:
first letter capitalize

Reply:
kebab-case doesn't use a capital letter at the start because the name is an example of the way to write in kebab-case

Reply:
forgot to mention that CamelCase was the only one which was capitalized and thats prettu funny :laughing: